packages/suite/src/views/wallet/tokens/common/UrlBlur.tsx

```diff
@@ -0,0 +1,35 @@
+import { Fragment } from 'react';
+import styled from 'styled-components';
+import { Tooltip } from '@trezor/components';
+
+import { BlurWrapper } from 'src/components/wallet/TransactionItem/TransactionItemBlurWrapper';
+import { Translation } from 'src/components/suite';
+
+const StyledTooltip = styled(Tooltip)`
+    display: inline-block;
+`;
+
+interface BlurUrlsProps {
+    text?: string;
+}
+
+export const BlurUrls = ({ text }: BlurUrlsProps) => {
+    if (!text) return null;
+
+    const urlRegex =
+        /\b(?:https?:\/\/|www\.)[a-zA-Z0-9-._~:/?#[\]@!$&'()*+,;=]+(?:\.[a-zA-Z]{2,})\b|[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)+/gi;
+
+    const parts = text.split(urlRegex);
+    const matches = [...text.matchAll(urlRegex)];
+
+    return parts.map((part, index) => (
```

Comment:
nit: would be nice to add here a comment of some example string

// Example string: Go to trezor.io
// parts is then this:
// matches is then this:
...

or take this logic to unil and create test